src/features/drop-manager/components/DropManager.tsx

```diff
@@ -38,6 +70,35 @@ export const DropManager = ({
     },
   ];
 
+  const handleExportCSVClick = () => {
+    if (data.length > 0) {
+      const links = data.map(({ dropLink }) => `${dropLink as string}`);
+      file(`Drop ID ${data[0].dropId as string}`, links.join('\r\n'));
```

Comment:
```suggestion
      file(`Drop ID ${data[0].dropId as string}.csv`, links.join('\r\n'));
```

currently exports a `.txt` file, shouldn't it export csv?

Reply:
maybe check the file helper for download too